src/main/java/carpet/mixins/CollectingNeighborUpdaterAccessor.java

```diff
@@ -0,0 +1,13 @@
+package carpet.mixins;
+
+import net.minecraft.world.level.redstone.CollectingNeighborUpdater;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(CollectingNeighborUpdater.class)
+public interface CollectingNeighborUpdaterAccessor {
+    @Accessor("count")
+    void setCount(int count);
+    @Accessor("maxChainedNeighborUpdates")
+    int getMaxChainedNeighborUpdates();
```

Comment:
Aren't the annotation parameters implicit by the method names? At the end of the day mixin will have to add it to the refmap anyway.